pkg/services/auth/auth_token.go

```diff
@@ -37,6 +37,14 @@ func (s *UserAuthTokenService) Init() error {
 	return nil
 }
 
+func IsBasicAuthEnabled() bool {
+	return setting.BasicAuthEnabled
+}
+
+func IsLoginDisabled() bool {
+	return setting.DisableLogin
+}
+
```

Comment:
I would suggest removing these. They don't belong in this package and using `setting.<property>` is preferable.